docs/asyncapi.yaml

```diff
@@ -127,42 +138,57 @@ components:
         expectedPaymentDate: "21.11.2025"
         adminName: "Jessica Lrrr"
       default: {}
+      minleght: 0
+      maxlenght: 100
```

Comment:
I don't think we need length here as this is describing an object

Reply:
Are we going to constraint this object in any way?

Reply:
we've already specified the mandatory fields on the object so that's all we need.

I don't see any reason to constrain it further, if teams want to send extra data, that's fine.